recipes/moments/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name="moments" %}
+{% set version="1.0.5" %}
+
+package:
+  name: {{ name }}
+  version: '{{ version }}'
```

Comment:
@apragsdale -- this line could be the problem?  In other recipes, I don't see the single quotes around the version.